### ISSUE TITLE
online compiler option docs do not match --help strings

### ISSUE THREAD

**Issue by mobileink** [ocaml/ocaml.org#446]
For example, for both compilers `--help` lists lots of options that are not mentioned on the website doc pages (e.g. [https://v2.ocaml.org/manual/comp.html](https://v2.ocaml.org/manual/comp.html).)

Suggestion: updating the website docs to match the help strings would be best, but at the least, the website docs should include a prominent mention that additional options are available and documented via `--help`.

**Comment by cuihtlauac** [comment]
Revisit: should we migrate this issue to ocaml/v2.ocaml.org and/or ocaml/ocaml?

**Comment by sabine** [comment]
Yes, that sounds like an upstream issue for the manual at ocaml/ocaml.
